### ISSUE TITLE
Can't start Selendroid session on device (0.13.0-SNAPSHOT)

### ISSUE THREAD

**Issue by dimkin-eu** [selendroid/selendroid#741]
I'm using Selendroid with java + TestNG and starting it using Intellij Idea
Selendroid dependencies are taken by maven
```
 <selendroid.version>0.13.0-SNAPSHOT</selendroid.version>
        <dependency>
            <groupId>io.selendroid</groupId>
            <version>${selendroid.version}</version>
            <artifactId>selendroid-standalone</artifactId>
        </dependency>
        <dependency>
            <groupId>io.selendroid</groupId>
            <version>${selendroid.version}</version>
            <artifactId>selendroid-client</artifactId>
        </dependency>
```

Firstly, Selendroid initialization was done like this
```
SelendroidConfiguration config = new SelendroidConfiguration();
config.setPort(4444);
config.setForceReinstall(true);
SelendroidLauncher selendroidLauncher = new SelendroidLauncher(config);
selendroidLauncher.launchSelendroid();
return new SelendroidDriver(DesiredCapabilities.android());
```
which causes errors, described in #675

So, I tried to change starting driver to
```
return new SelendroidDriver(new SelendroidCapabilities());
```
or
```
return new SelendroidDriver(new URL("http://localhost:4444/wd/hub"), new SelendroidCapabilities());
```

with no luck
```
Dec 10, 2014 9:13:26 AM io.selendroid.standalone.server.model.DeviceStore addEmulators
INFO: No emulators has been found.
Dec 10, 2014 9:13:26 AM io.selendroid.standalone.server.SelendroidStandaloneServer start
INFO: Selendroid standalone server has been started on port: 4444
Dec 10, 2014 9:13:26 AM io.selendroid.standalone.server.model.SelendroidStandaloneDriver createNewTestSession
SEVERE: Error occurred while starting Selendroid session
io.selendroid.server.common.exceptions.SessionNotCreatedException: The requested application under test is not configured in selendroid server.
	at io.selendroid.standalone.server.model.SelendroidStandaloneDriver.getAndroidApp(SelendroidStandaloneDriver.java:390)
	at io.selendroid.standalone.server.model.SelendroidStandaloneDriver.createNewTestSession(SelendroidStandaloneDriver.java:218)
	at io.selendroid.standalone.server.model.SelendroidStandaloneDriver.createNewTestSession(SelendroidStandaloneDriver.java:207)
	at io.selendroid.standalone.server.handler.CreateSessionHandler.handleRequest(CreateSessionHandler.java:40)
	at io.selendroid.standalone.server.BaseSelendroidStandaloneHandler.handle(BaseSelendroidStandaloneHandler.java:45)
	at io.selendroid.standalone.server.SelendroidServlet.handleRequest(SelendroidServlet.java:131)
	at io.selendroid.server.common.BaseServlet.handleHttpRequest(BaseServlet.java:67)
	at io.selendroid.server.common.http.ServerHandler.channelRead(ServerHandler.java:53)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:333)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:319)
	at io.netty.handler.traffic.AbstractTrafficShapingHandler.channelRead(AbstractTrafficShapingHandler.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:333)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:319)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:333)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:319)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:161)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:147)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:333)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:319)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:787)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:130)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:511)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:468)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:382)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:354)
	at io.netty.util.concurrent.SingleThreadEventExecutor$2.run(SingleThreadEventExecutor.java:116)
	at io.netty.util.concurrent.DefaultThreadFactory$DefaultRunnableDecorator.run(DefaultThreadFactory.java:137)
	at java.lang.Thread.run(Thread.java:745)
```

SelendroidCapabilities() returns such caps
```
"automationName" -> "selendroid"
"browserName" -> "selendroid"
"platformName" -> "android"
```

Any ideas ?

**Comment by mitchhentges** [comment]
Hey, just curious, is `http://localhost:4444/wd/hub/status` accessible _after_ driver initialization?
If it is, it should show you whether your app is being successfully picked up:
```json
{
    "value": {
        "os": {
            "name": "Windows 7",
            "arch": "amd64",
            "version": "6.1"
        },
        "build": {
            "browserName": "selendroid",
            "version": "0.13.0"
        },
        "supportedDevices": [
            {
                <stuff here>
            }
        ],
        "supportedApps": [
            {
                "mainActivity": "ca.derp.jook.dunks.Basketball",
                "appId": "ca.derp.jook.dunks:1234",
                "basePackage": "ca.derp.jook.dunks"
            },
            {
                "mainActivity": "io.selendroid.androiddriver.WebViewActivity",
                "appId": "io.selendroid.androiddriver:0.13.0",
                "basePackage": "io.selendroid.androiddriver"
            }
        ]
    },
    "status": 0
}
```

It looks like the important error is `The requested application under test is not configured in selendroid server.`, which makes sense as you haven't specified what app test to test:

```java
new SelendroidCapabilities("yourApp:yourAppVersion1234")
```

Sorry for the late out-of-nowhere response, but I had a similar issue as #675 but mine was solved by reinstalling my app on my tablet.

**Comment by dimkin-eu** [comment]
Hi, @mitchhentges . I'm not testing app - just web. 
In some lucky way with 0.14th version it tries to open Selendriod webview @ mob phone , but it dies immediately 

http://localhost:4444/wd/hub/status returning such info
```
{"value":{"os":{"name":"Windows 7","arch":"amd64","version":"6.1"},"build":{"browserName":"selendroid","version":"0.14.0"},"supportedDevices":[{"emulator":false,"screenSize":"(1080, 1920)","serial":"8fdf99cf","platformVersion":"19","model":"GT-I9515"}],"supportedApps":[{"mainActivity":"io.selendroid.androiddriver.WebViewActivity","appId":"io.selendroid.androiddriver:0.14.0","basePackage":"io.selendroid.androiddriver"}]},"status":0}
```

**Comment by dimkin-eu** [comment]
probably this could be closed because of #813

**Comment by asm89** [comment]
I'm closing this issue due to inactivity. If you still experience the issue described on the latest release of selendroid feel free to file a new issue. If you do please provide a [concise reproducible test case](http://sscce.org/) and describe what results you are seeing and what results you expect.
